src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -8,4 +10,6 @@
 
 	Optional<User> findByUsername(String username);
 
+	List<User> findByBirthdateBetweenAndNameIgnoreCase(LocalDate startDate, LocalDate endDate,String name);
```

Comment:
According to the card the attribute name must contain a character sequence and not be equals, you can replace the method to **findByBirthdateBetweenAndNameContainingIgnoreCase**

Reply:
done